commandsv3/src/main/java/org/wpilib/commands3/Mechanism.java

```diff
@@ -63,8 +37,8 @@ public Mechanism(String name, Scheduler scheduler) {
    * @return The name of the mechanism.
    */
   @NoDiscard
-  public String getName() {
-    return m_name;
+  default String getName() {
+    return this.getClass().getSimpleName();
```

Comment:
This behavior should be documented in the javadoc. The `this` keyword is also unnecessary here.